python/lsst/ts/fbs/utils/data/comcam_field_survey_centers.yaml

```diff
@@ -1,4 +1,12 @@
-comcam_sv_targets:
+sv_targets:
```

Comment:
I think with the file renaming here you are considering "archiving" the comcam_field_survey_centers.yaml file, which is fine, but I'm not sure why you renamed the target class from `comcam_sv_targets` to `sv_targets` here. Was it for backwards compatibility? I think if you remove the `sv_targets` filter in the method (see comment below) you won't need to do this.

Just trying to avoid having any sets of `sv_targets` in two different place (here and ts_config_ocs).

Reply:
Good catch. I think it would be preferable to remove this file altogether to avoid confusion.